android/java/org/chromium/chrome/browser/crypto_wallet/activities/BraveWalletBaseActivity.java

```diff
@@ -236,7 +235,6 @@ public void finishNativeInitialization() {
     public void onDestroy() {
         if (mKeyringServiceObserver != null) {
             mKeyringServiceObserver.close();
-            mKeyringServiceObserver.destroy();
```

Comment:
why that file been changed? I would propose to keep PR as small as possible and remove `destroy` if it's totally unnecessary when you change something inside `BraveWalletBaseActivity`.

Reply:
I see you replace to `WeakRef` and deleted that method.